bidadjustment/build_rules.go

```diff
@@ -63,7 +63,7 @@ func merge(req *openrtb_ext.RequestWrapper, acct *openrtb_ext.ExtRequestPrebidBi
 	if extPrebid == nil && acct == nil {
 		return nil, nil
 	}
-	if extPrebid == nil && acct != nil {
+	if (extPrebid == nil || extPrebid.BidAdjustments == nil) && acct != nil {
```

Comment:
Please add a test covering this panic condition, so we can confirm this fix is all that's necessary.